geonode/security/views.py

```diff
@@ -38,8 +38,9 @@
 from geonode.layers.models import Layer
 from geonode.groups.models import GroupProfile
 
-if "notification" in settings.INSTALLED_APPS:
-    from notification import models as notification
+from geonode.notifications_helper import (send_notification, queue_notification,
+                                          has_notifications, get_notification_recipients,
+                                          get_comment_notification_recipients)
```

Comment:
Are those additional imports needed also?

Reply:
sorry, no they are not. I just copied the import line from another file.